src/services/db.ts

```diff
@@ -0,0 +1,14 @@
+import { DB_URL } from "../constants";
+
+const MongoClient = require("mongodb").MongoClient;
```

Comment:
Still can't understand, why are using `require` in typescript project :disappointed: